posts/2023/03/16/kubernetes-meets-event-driven-ansible.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Kubernetes Meets Event-Driven Ansible
+author: Andrew Block
+date: 2023-03-16 13:00:00
+slug: kubernetes-meets-event-driven-ansible
+category: event-driven-ansible
+tags: event-driven-ansible, kubernetes, rulebooks, eda, ansible-rulebook
+type: text
+---
+In today’s fast paced world, every second counts and the ability to react to activities in a timely fashion can mean the difference between satisfying the needs of consumers and meeting Service-Level Agreements. Each are goals of [Event-Driven Ansible](https://www.ansible.com/use-cases/event-driven-automation), which seeks to further the reach of Ansible based automation by responding to events that meet certain criteria. These events can originate from a variety of sources, such as from an HTTP endpoint, messages on a queue or topic, or from public cloud resources. Kubernetes has become synonymous with managing infrastructure and applications in cloud native architectures and many organizations are reliant on these systems for running their business critical workloads. Automation and Kubernetes go hand in hand and Ansible already plays a role within this ecosystem. A new capability leveraging the Event-Driven Ansible framework is now available that extends the integration between both Ansible and Kubernetes so that Ansible automation activities can be triggered based on events and actions occurring within a Kubernetes cluster.
```

Comment:
```suggestion
In today's fast paced world, every second counts and the ability to react to activities in a timely fashion can mean the difference between satisfying the needs of consumers and meeting Service-Level Agreements. Each are goals of [Event-Driven Ansible](https://www.ansible.com/use-cases/event-driven-automation), which seeks to further the reach of Ansible based automation by responding to events that meet certain criteria. These events can originate from a variety of sources, such as from an HTTP endpoint, messages on a queue or topic, or from public cloud resources. Kubernetes has become synonymous with managing infrastructure and applications in cloud native architectures and many organizations are reliant on these systems for running their business critical workloads. Automation and Kubernetes go hand in hand and Ansible already plays a role within this ecosystem. A new capability leveraging the Event-Driven Ansible framework is now available that extends the integration between both Ansible and Kubernetes so that Ansible automation activities can be triggered based on events and actions occurring within a Kubernetes cluster.
```